GitUpKit/Core/GCLiveRepository.m

```diff
@@ -1008,8 +1008,18 @@ static BOOL _MatchReference(NSString* match, NSString* name) {
 - (NSArray*)findCommitsMatching:(NSString*)match {
   XLOG_DEBUG_CHECK(_database);
   NSMutableArray* results = [[NSMutableArray alloc] init];
+  
   match = [match stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]];
-  if (match.length >= kMinSearchLength) {
+  bool searchFileHistoryOnly = [match hasPrefix:@"/"];
+  // Search file history directly
+  if(match.length >= kMinSearchLength && searchFileHistoryOnly) {
```

Comment:
This needs to be (kMinSearchLength + 1) because of prefix /

Reply:
Will change it in next commit.